gcp_variant_transforms/libs/annotation/annotation_parser.py

```diff
@@ -43,6 +43,9 @@
 _BREAKEND_ALT_RE = (re.compile(
     r'^(?P<up_to_chr>.*([\[\]]).*):(?P<pos>.*)([\[\]]).*$'))
 
+# Filled with annotation field and name data, then used as a header ID.
```

Comment:
I prefer to still keep these in `infer_headers_util.py`.
This module is used to parse/reconstruct the existing annotation header (something like `A|upstream_gene_variant|MODIFIER|PSMF1|||||`), which is usually added by the third party tool.

The new ` get_inferred_annotation_type_header_key`, creates a name for each individual annotation field when `infer_annotation_type` is set. For instance, we may add something like below into one temporary header file to help loading the VCF to BQ.
`##INFO=<ID=CSQ_VT_SWISSPROT_TYPE,Number=1,Type=String,Description="Inferred type field for annotation SWISSPROT.",Source="",Version="">`